sample-app/src/main/java/com/joinforage/android/example/ui/pos/screens/shared/PinEntryScreen.kt

```diff
@@ -22,6 +30,13 @@ fun PINEntryScreen(
     ScreenWithBottomRow(
         mainContent = {
             if (paymentMethodRef != null) {
+                Card {
+                    Column(modifier = Modifier.padding(16.dp)) {
+                        Text("Payment Method", fontWeight = FontWeight.SemiBold)
+                        Text("Ref: " + paymentMethodRef)
```

Comment:
These details are very helpful! Can we also display the `Refund.ref` if its a `Refund` flow? That should not block this PR but I wonder if that could be helpful to show `Payment.ref` / `Refund.ref` defpending on the flow